src/util/metrics/threads_linux.rs

```diff
@@ -415,26 +376,6 @@ mod tests {
         assert!(get_thread_name("invalid_stat").is_err());
     }
 
```

Comment:
Please add a test to test `Stat::collect`.

Reply:
There're existing tests not removed -- `test_thread_stat` which calls `Stat::collect()`.

I think I should add some assertions. Will impl soon